throttle.py

```
@@ -0,0 +1,29 @@
+"""
+Utility module for rate limiting requests.
+"""
+import time
+from typing import Optional
```

Ruff (0.8.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)